### ISSUE TITLE
No response in store activity if Karma points are zero.

### ISSUE THREAD

**Issue by pknotfound** [systers/powerup-android#1410]
## Description
There is no feedback if karma points are zero in the StoreActivity

## Mocks

![WhatsApp Video 2020-03-20 at 1 34 49 AM](https://user-images.githubusercontent.com/52661249/77111034-98182780-6a4c-11ea-994c-b4149867bbb6.gif)

## Update [Required]
The user should be shown a prompt that they do not have karma points left.

**Comment by pknotfound** [comment]
@m-murad May I work on this issue?
Although this kind of issue has been raised [before](https://github.com/anitab-org/powerup-android/issues/1014#issue-300028728) and was closed.

I think that graying the items is not quite clear and the user should be explicitly told about the Karma points.

Please review.

**Comment by sammy1997** [comment]
I agree with @Priyansh-Kedia  here. This creates a bad UX. We should add a toast or snackbar or a dialog. @m-murad @isabelcosta views?

**Comment by HaripriyaB** [comment]
@Priyansh-Kedia are u available to work on this issue?

**Comment by pknotfound** [comment]
Yes @HaripriyaB 
I am still waiting for approval

**Comment by HaripriyaB** [comment]
Okay assigning it to you. @Priyansh-Kedia Can u discuss what exactly is your approach for solving this?

**Comment by pknotfound** [comment]
I was thinking of showing a toast or something for the same.
Still waiting for suggestions as @sammy1997 suggested.

**Comment by HaripriyaB** [comment]
A custom toast with a good UI would be better. @Priyansh-Kedia I think u can start working and show the mockups here so that we can decide which one is better.
